src/pspm_init.m

```diff
@@ -192,285 +203,285 @@
-    'import', @none, ...
-    'data', 'wave');
+  struct('type', 'pupil_missing_r', ...
+  'description', 'Pupil data missing/interpolated right', ...
+  'import', @none, ...
+  'data', 'wave');
 %add channel for blink left
 defaults.chantypes(26) = ...
-    struct('type', 'blink_l', ...
-    'description', 'Blink left', ...
-    'import', @pspm_get_blink_l, ...
-    'data', 'wave');
+  struct('type', 'blink_l', ...
+  'description', 'Blink left', ...
+  'import', @pspm_get_blink_l, ...
+  'data', 'wave');
 
 %add channel for blink right
 defaults.chantypes(27) = ...
-    struct('type', 'blink_r', ...
-    'description', 'Blink right', ...
-    'import', @pspm_get_blink_r, ...
-    'data', 'wave');
+  struct('type', 'blink_r', ...
+  'description', 'Blink right', ...
+  'import', @pspm_get_blink_r, ...
+  'data', 'wave');
 
 %add channel for saccade left
 defaults.chantypes(28) = ...
-    struct('type', 'saccade_l', ...
-    'description', 'Saccade left', ...
-    'import', @pspm_get_saccade_l, ...
-    'data', 'wave');
+  struct('type', 'saccade_l', ...
+  'description', 'Saccade left', ...
+  'import', @pspm_get_saccade_l, ...
+  'data', 'wave');
 
 %add channel for saccade right
 defaults.chantypes(29) = ...
-    struct('type', 'saccade_r', ...
-    'description', 'Saccade right', ...
-    'import', @pspm_get_saccade_r, ...
-    'data', 'wave');
+  struct('type', 'saccade_r', ...
+  'description', 'Saccade right', ...
+  'import', @pspm_get_saccade_r, ...
+  'data', 'wave');
 
-%add channel for (gaze) scanpath speed  
+%add channel for (gaze) scanpath speed
 defaults.chantypes(30) = ...
-    struct('type', 'sps', ...
-    'description', 'Scanpath speed', ...
-    'import', @pspm_get_sps, ...
-    'data', 'wave');
+  struct('type', 'sps', ...
+  'description', 'Scanpath speed', ...
+  'import', @pspm_get_sps, ...
+  'data', 'wave');
 
 defaults.chantypes(31) = ...
-    struct('type', 'sps_r', ...
-    'description', 'Scanpath speed', ...
-    'import', @pspm_get_sps_r, ...
-    'data', 'wave');
+  struct('type', 'sps_r', ...
+  'description', 'Scanpath speed', ...
+  'import', @pspm_get_sps_r, ...
+  'data', 'wave');
 
 defaults.chantypes(32) = ...
-    struct('type', 'sps_l', ...
-    'description', 'Scanpath speed', ...
-    'import', @pspm_get_sps_l, ...
-    'data', 'wave');
+  struct('type', 'sps_l', ...
+  'description', 'Scanpath speed', ...
+  'import', @pspm_get_sps_l, ...
+  'data', 'wave');
 
 % documented
 defaults.chantypes(33) = ...
-    struct('type', 'custom', ...
-    'description', 'Custom', ...
-    'import', @pspm_get_custom, ...
-    'data', 'wave');
+  struct('type', 'custom', ...
+  'description', 'Custom', ...
+  'import', @pspm_get_custom, ...
+  'data', 'wave');
 defaults.chantypes(34) = ...
-    struct('type', 'pupil_r_pp', ...
-    'description', 'Pupil right preprocessed', ...
-    'import', @none, ...
-    'data', 'wave');
+  struct('type', 'pupil_r_pp', ...

```

Comment:
@SerenXia Sorry, I do not recommend to write it like `pupil_r_pp`, instead, write it as `pupil_pp_r`, because some code assume the "side marker" is written at the end, for example in line 331--332 in `pspm_find_valid_fixations`.
> `channels(str_chans) = regexprep(channels(str_chans), '(pupil_pp|gaze_x|gaze_y|pupil_missing)', ['$0_' eye]);`

Reply:
> @SerenXia Sorry, I do not recommend to write it like `pupil_r_pp`, instead, write it as `pupil_pp_r`, because some code assume the "side marker" is written at the end, for example in line 331--332 in `pspm_find_valid_fixations`.
> 
> > `channels(str_chans) = regexprep(channels(str_chans), '(pupil_pp|gaze_x|gaze_y|pupil_missing)', ['$0_' eye]);`

Hi Teddy, this naming format was not determined by me. I don't know who determined it, but if you want to change them (as in many related functions they are named in this way), you may need to discuss with Dominik.

Best regards,
Yanfang

Reply:
> > @SerenXia Sorry, I do not recommend to write it like `pupil_r_pp`, instead, write it as `pupil_pp_r`, because some code assume the "side marker" is written at the end, for example in line 331--332 in `pspm_find_valid_fixations`.
> > > `channels(str_chans) = regexprep(channels(str_chans), '(pupil_pp|gaze_x|gaze_y|pupil_missing)', ['$0_' eye]);`
> 
> Hi Teddy, this naming format was not determined by me. I don't know who determined it, but if you want to change them (as in many related functions they are named in this way), you may need to discuss with Dominik.
> 
> Best regards,
> Yanfang

Exactly what I am going to do!
Thanks Yanfang.

Best
Teddy